packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -401,6 +401,23 @@ export const timm = (model: ModelData): string[] => [
 model = timm.create_model("hf_hub:${model.id}", pretrained=True)`,
 ];
 
+export const seed_story = (): string[] => [
+	`# Install from https://github.com/TencentARC/SEED-Story.git
+# Use in SEED-Story (https://huggingface.co/TencentARC/SEED-Story)
```

Comment:
Can be removed imo
1. There will be a link to the GitHub repo too
2. The snippet will only show in the SEED-Story repo

```suggestion
```